templates/taxi/manufacturer_list.html

```diff
@@ -1,7 +1,7 @@
 {% extends "base.html" %}
 
 {% block content %}
-    <h1>Manufacturer List
+    <h1>Manufacturer List <a href="{% url "taxi:manufacturer-create" %}">+</a>
```

Comment:
Consider adding Bootstrap classes to the '+' link to make it more visually appealing and consistent with other buttons or links in the application, such as `class="btn btn-primary"`.